docs/x509/ocsp.rst

```diff
@@ -690,3 +707,71 @@ Interfaces
 
         Encode the X.509 ``Name`` of the certificate whose private key signed
         the response.
+
+.. class:: OCSPResponseIterator
+
+    ..versionadded:: 37.0.0
+
+    An iterator working through each SINGLERESP structure attached
+    to an OCSP response.
+
+        .. attribute:: certificate_status
+
+        :type: :class:`~cryptography.x509.ocsp.OCSPCertStatus`
```

Comment:
```suggestion
    :type: :class:`~cryptography.x509.ocsp.OCSPCertStatus`
```

Reply:
Looking at everything else, I moved the `.. attribute` line up one indent-level.  That gets all the `.. attribute`s to line up with the others, and the `:type:` lines to line up with the others.  Hopefully I didn't screw that up.